### ISSUE TITLE
Feature Request: Append Docker labels as tags in dogstatsd

### ISSUE THREAD

**Issue by ejholmes** [DataDog/dd-agent#2497]
One of the really nice features of the datadog agent is being able to automatically tag all of the Docker metrics with a whitelist of labels. It would be nice if this feature extended to dogstatsd, so that any metrics submitted from a Docker container, also get the labels appended to their metrics.

I realize this is by no means a trivial feature request, but it would be immensely useful.

**Comment by remh** [comment]
@ejholmes 
Yeah I completely agreed. This use case makes complete sense and we are discussing of it internally. 

As you said this is not a trivial feature as we need to consolidate the tags coming from multiple sources. We have a few ideas in mind and we'll start working on that soon. 

Thanks for the feedback.
